fontir/src/error.rs

```diff
@@ -16,6 +16,10 @@ pub enum Error {
     IoError(#[from] io::Error),
     #[error("Unable to parse {0:?}: {1}")]
     ParseError(PathBuf, String),
+    #[error("No axes are defined for master {0}")]
```

Comment:
"No _axis values_ are defined for master {0}", instead of simply "axes".

Also this is looks like a very Glyphs specific error, not even sure if possible to trigger at all with a valid source file.

By the way, in designspace, sources' locations may omit some or even all dimensions and these will be filled in with the default location (in design-space coords) across all globally-defined axes.

https://fonttools.readthedocs.io/en/latest/designspaceLib/python.html#fontTools.designspaceLib.SourceDescriptor.getFullDesignLocation

Reply:
however because in Glyphs.app there are no global axes definitions with user-space min/default/max (only a list of axes names and tags), we can't infer a default master location in this case. In fact we go the opposite way and determine the global axes min/default/max from the masters locations.
I think in Glyphs we should just assume (assert) masters will have as many axis values as there are axes, like you're already doing with InconsistentAxisDefinitions error.

Reply:
Fixed message